.github/workflows/maven-build-all.yml

```diff
@@ -31,7 +31,8 @@ jobs:
       run: mvn -B package --file aws-lambda-java-events-sdk-transformer/pom.xml
     - name: Build log4j2 with Maven
       run: mvn -B package --file aws-lambda-java-log4j2/pom.xml
-    - name: Build serialization with Maven
-      run: mvn -B package --file aws-lambda-java-serialization/pom.xml
-    - name: Build runtime-interface-client with Maven
-      run: mvn -B package --file aws-lambda-java-runtime-interface-client/pom.xml
+
+    # Test Runtime Interface Client
+    - name: Run 'pr' target
+      working-directory: ./aws-lambda-java-runtime-interface-client
+      run: make pr
```

Comment:
Looks like this actually takes a while - I suggest we create a separate workflow that would only trigger if changes were made for the RIC or any of its dependent libraries (`aws-lambda-java-core` and `aws-lambda-java-serialization`).

ref. https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-including-paths

Reply:
We could even take the opportunity to define a workflow per package as there is probably no need to build them all on every change.

Reply:
Creating a follow up PR for restructuring the workflow.

Reply:
Sounds good!